src/06-errors_and_files/03-exceptions.md

```diff
@@ -129,6 +129,7 @@ the `Exception` type class has; I think
 ```hs
 -- need to add these at the top
 
+{-# language LambdaCase #-}
```

Comment:
I thought it would make it complete because here you have the whole module and also use 'add these' ... as plural

Reply:
By these I was referring to the new language extension and the import, since the module isn't actually complete - we are missing some imports and the data declaration from previous snippets.

Reply:
ok